DOCS-7173: CSM setup docs reorg (#21413)
DataDog/documentation: content/en/security/cloud_security_management/setup/csm_cloud_workload_security/agent/kubernetes.md

---
title: Enabling CSM Cloud Workload Security on Kubernetes
kind: documentation
code_lang: kubernetes
type: multi-code-lang
code_lang_weight: 60 # a number that represents relative weight. 
---

Use the following instructions to enable [CSM Threats][1] on Kubernetes. To learn more about the supported deployment types for each CSM feature, see [Setting Up Cloud Security Management][2].

{{< tabs >}}

> {{% tab "Operator" %}}

Reviewer: ```suggestion
{{% tab "Datadog Operator" %}}
```

just for extra context

> 1. Add the following to the `spec` section of the `values.yaml` file:

Reviewer: ```suggestion
1. Add the following to the `spec` section of the `datadog-agent.yaml` file:
```

the operator spec file is called datadog-agent.yaml

    ```yaml

>     # values.yaml file

Reviewer: ```suggestion
    # datadog-agent.yaml file
```

    spec:
      features:
        remoteConfiguration:
          enabled: true
        cws:
          enabled: true
    ```

> 2. Restart the Agent.

Reviewer 2: I think technically it's more correct to say `Apply the changes` then restart the agent in the case of k8s Operator installation

[2]: https://github.com/DataDog/datadog-operator/blob/main/docs/configuration.v2alpha1.md

{{% /tab %}}

{{% tab "Helm" %}}

> 1. Add the following to the `datadog` section of the `values.yaml` file:

Reviewer: ```suggestion
1. Add the following to the `datadog` section of the `datadog-values.yaml` file:
```

container docs generally refer to the custom helm config file as `datadog-values.yaml`—the `values.yaml` file is the default helm chart config, where parameters not set in `datadog-values.yaml` default to those in `values.yaml`

Author: Didn't realize that. That's good to know! Thanks for the heads up.

    ```yaml

>     # values.yaml file

Reviewer: ```suggestion
    # datadog-values.yaml file
```

    datadog:
      remoteConfiguration:
        enabled: true
      securityAgent:
        runtime:
          enabled: true
    ```

2. Restart the Agent.

{{% /tab %}}

> {{% tab "Daemonset" %}}

Reviewer: ```suggestion
{{% tab "DaemonSet" %}}
```
